site/_docs/reference.md

```diff
@@ -2752,6 +2752,8 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | FROM_BASE32(string)                            | Returns the decoded result of a base-32 *string* as a string
 | m | TO_BASE64(string)                              | Converts the *string* to base-64 encoded form and returns a encoded string
 | b m | FROM_BASE64(string)                          | Returns the decoded result of a base-64 *string* as a string
+| b | TO_HEX(binary)                                 | Converts *binary* into a hexadecimal varchar
+| b | FROM_HEX(varchar)                              | Converts a hexadecimal-encoded *varchar* into bytes
```

Comment:
why `VARCHAR` and not `STRING`?

Reply:
Should we use types in the related libs or in calcite? I think it is the latter.

Reply:
Agree with Dan, we should use Calcite's type names when describing the func.

Reply:
That makes sense to me, do you know why many of the functions use `string` in their documentation?

Reply:
@tanclary IMO, people can easily confuse `string` and `varchar`, as we described in [CALCITE-5777](https://issues.apache.org/jira/projects/CALCITE/issues/CALCITE-5777?filter=myopenissues). 
STRING_TYPES not only contains CHAR_TYPES, but also BINARY_TYPES.
`public static final List<SqlTypeName> STRING_TYPES =
      combine(CHAR_TYPES, BINARY_TYPES);`

Reply:
That makes sense, thanks for explaining